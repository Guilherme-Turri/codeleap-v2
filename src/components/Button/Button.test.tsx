import { Button } from './'
import { componentRender } from '../../render/componentRender'
import { screen } from '@testing-library/react'

describe('<Button />', () => {
  it('Should render on screen properly', () => {
    const { container } = componentRender(<Button>test</Button>);
    expect(screen.getByText('test')).toBeInTheDocument();
    expect(container).toMatchSnapshot()
  })
})