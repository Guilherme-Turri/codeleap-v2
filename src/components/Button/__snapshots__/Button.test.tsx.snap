// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Should render on screen properly 1`] = `
.c0 {
  all: unset;
  background-color: #00CBA9;
  border-radius: 0.313rem;
  width: 100%;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
}

<div>
  <button
    class="c0"
  >
    test
  </button>
</div>
`;
